src/channel.cr

```diff
@@ -114,6 +114,10 @@ class Channel(T)
     end
   end
 
+  # Closes the channel.
+  # The method prevents any new value from being sent to / received from the channel.
+  # It wakes up any sender / receiver fibers waiting on the channel.
```

Comment:
All fibers blocked in `send` or `receive` will recieve `Channel::ClosedError`?

Reply:
Exactly, see
* [spec 1](https://github.com/crystal-lang/crystal/blob/master/spec/std/channel_spec.cr#L76) and [spec 2](https://github.com/crystal-lang/crystal/blob/master/spec/std/channel_spec.cr#L449) for `#receive`
* [spec](https://github.com/crystal-lang/crystal/blob/master/spec/std/channel_spec.cr#L143) for `#receive?`
* [spec 1](https://github.com/crystal-lang/crystal/blob/master/spec/std/channel_spec.cr#L217) and [spec 2](https://github.com/crystal-lang/crystal/blob/master/spec/std/channel_spec.cr#L427) for `#send`

Reply:
@RX14 
> All fibers blocked in send or receive will receive Channel::ClosedError

is ambiguous in that one might expect to actually `receive` an object of type ClosedError. I'm opting for

> All fibers blocked in send or receive will be awakened with Channel::ClosedError 

Let me know what you think